### PR TITLE
fix: fix copilot plugin api-key&oauth template

### PR DESCRIPTION
**Pull request by Yimin-Jin**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/28484156

### CODE REVIEW

File: templates/csharp/api-plugin-from-scratch-bearer/.{{NewProjectTypeName}}/README.md.tpl [hunk 1]
```diff
@@ -9,14 +9,30 @@
...
-3. If prompted, sign in with a Microsoft 365 account for the Teams organization you want to install the app to.
-4. Press F5, or select the `Debug > Start Debugging` menu in Visual Studio to start your app
+3. Right-click the '{{NewProjectTypeName}}' project and select Teams Toolkit > Prepare Teams App Dependencies
+4. If prompted, sign in with a Microsoft 365 account for the Teams organization you want to install the app to.
+5. Press F5, or select the `Debug > Start Debugging` menu in Visual Studio to start your app
 </br>![image](https://raw.githubusercontent.com/OfficeDev/TeamsFx/dev/docs/images/visualstudio/debug/debug-button.png)
-5. When Teams launches in the browser, you can open the Copilot app and send a prompt to trigger your plugin.
-6. Send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
+6. When Teams launches in the browser, you can open the Copilot app and send a prompt to trigger your plugin.
+7. Send a message to Copilot to find an NuGet package information. For example: Find the NuGet package info on Microsoft.CSharp.
```
Comment (huimiu) [inline templates/csharp/api-plugin-from-scratch-bearer/.{{NewProjectTypeName}}/README.md.tpl:35]: This template is not for finding an NuGet package, maybe you should modify this part to:

```
6. When Teams launches in the browser, click the Apps icon from Teams client left rail to open Teams app store and search for Copilot.
7. Open the `Copilot` app, select `Plugins`, and from the list of plugins, turn on the toggle for your plugin. Now, you can send a prompt to trigger your plugin.
8. Send a message to Copilot to query the repair record. For example: List all repairs.
```

Comment (Yimin-Jin) [reply templates/csharp/api-plugin-from-scratch-bearer/.{{NewProjectTypeName}}/README.md.tpl:35]: Sure.

**Review by huimiu — APPROVED** [review]

File: templates/csharp/api-plugin-from-scratch-bearer/Repair.cs.tpl [hunk 2]
```diff
@@ -2,10 +2,11 @@ using Microsoft.Azure.Functions.Worker;
 using Microsoft.Azure.Functions.Worker.Http;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.Logging;
+using System.Net;
```
Comment (eriolchan) [inline templates/csharp/api-plugin-from-scratch-bearer/Repair.cs.tpl:5]: It looks like change class/variable name, why does it need to include new library of System.Net?

Comment (eriolchan) [reply templates/csharp/api-plugin-from-scratch-bearer/Repair.cs.tpl:5]: Why this class is Repair and that in templates/csharp/api-plugin-from-scratch-oauth/Repairs.cs.tpl is Repairs, can we keep them aligned?

Comment (Yimin-Jin) [reply templates/csharp/api-plugin-from-scratch-bearer/Repair.cs.tpl:5]: If we don't have this new library, it will cause HttpStatusCode not to be found.

**Review by huimiu — APPROVED** [review]

**Pull request merged**